Tools/test_keys/rsa2048.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Should there really be a private key committed to the repo? This doesn't quite seem right. Could the private key be generated before a test?

Reply:
This is clearly stated that this is a "test key", which is ment only for testing the crypto functionality. Also for example image building gives a clear warning every time when someone signs the image with this key.

Generating a key for simple test purpose adds the pain for e.g. adding the corresponding public key every time to your key storage for bootloader etc.

So this is sort of required for easy testing of the code when developing, in CI and on the desk...

Reply:
Ok fair enough.